### ISSUE TITLE
Feature request: Autohide buffer tabs after user-specified time

### ISSUE THREAD

**Issue by uolot** [tomiis4/BufferTabs.nvim#6]
Cool plugin, looks nice, thanks for creating it!

It would be great if it was possible to specify a timeout for hiding the buffer tabs. I don't want to always see the tabs but it makes a lot of sense for them to show up when I navigate between splits. The existing toggle command is nice, but of course I wouldn't want to keep constantly toggling tabs on and off :) 

If you're busy, I'm willing to look into it and open a PR, just let me know if you're interested.

**Comment by tomiis4** [comment]
Hi and sorry for the late response.

It sounds like a good idea, I will try to implement this week. Thank you so much for using this plugin.

**Comment by tomiis4** [comment]
It should be done, thank you so much.

**Comment by uolot** [comment]
Amazing, thank you for implementing it so quickly!

I've played with it a little bit and I noticed some erratic behaviour. If you set a bit longer timeout, like 5000ms, the tabs sometimes disappear after much shorter time. To reproduce, just open two splits and quickly navigate between them back and forth.

This seems to be caused by the previously scheduled function being triggered even if new timers has been scheduled since.

I've made a small PR that seems to fix it: https://github.com/tomiis4/BufferTabs.nvim/pull/7
